i18n/en.i18n.json

```diff
@@ -200,6 +200,7 @@
   "Room" : "Room",
   "Room_deleted" : "Room deleted",
   "Room_deleted_by" : "Room deleted by: <em>__user_by__</em>",
+  "Room_label_changed" : "Security label changed to: <em>__label__</em> by: <em>__user_by__</em>",
```

Comment:
What if you hack it by changing:
> <em>__label__</em>

to:
> <em style=\"text-transform:uppercase\">__label__</em>

Not ideal, but it works, and this particular case is kind of an exception.